vike-react/renderer/onRenderHtml.tsx

```diff
@@ -33,32 +33,33 @@ const onRenderHtml: OnRenderHtmlAsync = async (pageContext): ReturnType<OnRender
     </React.StrictMode>
   )
 
+  const headHtml = dangerouslySkipEscape(renderToString(head))
+
   const isSsrDisabled = !pageContext.Page
   const page = isSsrDisabled ? <></> : getPageElement(pageContext)
-  const htmlContent = (
-    <>
+
+  // Which one is better ?
+  // This :
+  const streamOrString = await renderToStream(page, { userAgent: pageContext.userAgent, disable: isSsrDisabled })
+  /**
+   * Or This ?
+  const streamOrString = isSsrDisabled ? dangerouslySkipEscape(renderToString(page)) : await renderToStream(page, { userAgent: pageContext.userAgent })
```

Comment:
This one is better: we don't want to use `renderToStream()` when SSR is disabled, see https://github.com/vikejs/vike-react/pull/34